bve/src/datatypes.rs

```diff
@@ -1,10 +1,38 @@
 use cgmath::{Vector1, Vector2, Vector3, Vector4};
 
+pub trait Asu32 {
```

Comment:
Why did you do this instead of implementing the `TryFrom` or `TryInto` traits?

Reply:
Because it changes the semantic meaning of the thing. `From` and friends are there to convert a value from one form to another. I'm basically reinterpreting here, and the result is not semantically equivalent. Just converting from a `Vector4<u8>` to a `u32` is meaningless (x? y? z? w? sum? avg?) so it deserves a different trait.

Reply:
I disagree but I respect your opinion